collectors/python.d.plugin/slurm/README.md

```diff
@@ -0,0 +1,21 @@
+<!--
+title: "Slurm queue monitoring with Netdata"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/python.d.plugin/slurm/README.md
+sidebar_label: "Slurm Queue"
+-->
+
+# Slurm queue monitoring with Netdata
+
+Monitors slurm queue statistics using the squeue tool.  
+
+Execute `squeue` to grab information from the slurm queue.
+
+## Configuring slurm
+
+Slurm monitoring is enabled by default. 
+
```

Comment:
```suggestion
### Example

```yaml
# job_name:
#     name: myname            # the JOB's name as it will appear at the
#                             # dashboard (by default is the job_name)
#                             # JOBs sharing a name are mutually exclusive
#     update_every: 1         # the JOB's data collection frequency
#     priority: 60000         # the JOB's order on the dashboard
#     penalty: yes            # the JOB's penalty
#     autodetection_retry: 0  # the JOB's re-check interval in seconds
```
```

Reply:
Out of curiosity: is this a slurm-specific configuration?

Reply:
This is from the configuration file you submitted. :)

Reply:
If you want to, please submit a hands-on example.